### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by jsageryd**

### CODE REVIEW

File: 2-basics/presentation.slide [hunk 1]
```diff
@@ -135,7 +135,7 @@ Ampersand (`&`) is used to fetch a pointer to an existing value.
 
 .code pointers/example1.go /START REF OMIT/,/END REF OMIT/
 
-Asterisk (`*`) is used to fetch, or _dereference_, the value a pointer points to.
+Asterisk (`*`) is used to _dereference_ a pointer to retreive the value.
```
Comment (marcusolsson) [inline 2-basics/presentation.slide]: I believe it's supposed to say _retrieve_ here?

**Comment by marcusolsson** [comment]
Well, this is a rather embarrassing PR to read. I left one minor comment though, feel free to  or I'll merge and fix it later today.

File: 2-basics/presentation.slide (hunk 1 above)
Comment (jsageryd) [inline 2-basics/presentation.slide]: I _always_ misspell this word. I don't know why. Thanks :) Fixing and force-pushing.

**Comment by jsageryd** [comment]
Fixed. And please don't feel embarrassed; it's not like these things are major in any way.

**Comment by marcusolsson** [comment]
Thank you!

**Pull request merged**